dev-python/pipenv/pipenv-2022.4.8.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
Please use newer [PEP517 syntax](https://projects.gentoo.org/python/guide/distutils.html#basic-use-pep-517-mode)

Reply:
Is this instead of `DISTUTILS_USE_SETUPTOOLS=rdepend` or in addition?

Reply:
instead

Reply:
Thanks!